memory.go

```diff
@@ -11,9 +11,19 @@ import (
 	"math"
 )
 
+type MemoryBank struct {
```

Comment:
Let's call this `MemoryModule`, to align with industry terminology:

https://en.wikipedia.org/wiki/Memory_module

I know CMI/WMI uses the term "bank", but generally these things are referred to as "modules" in my experience :)